action.yml

```diff
@@ -16,6 +17,22 @@ inputs:
     description: 'Limit the number of first x lines from commit messages that will be added in the issue description. No truncation when set to 0.'
     required: false
     default: 1
+
 runs:
-  using: 'node16'
-  main: 'dist/index.js'
+  using: 'composite'
+  steps:
+    - name: Checkout the fresh project
+      uses: actions/checkout@v3
+
+    - name: Fetch the tags
+      run: |
+        git fetch
+        git fetch --tags
+      shell: bash
```

Comment:
this can more simply be done in the previous step by doing this https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches